back-end/index.js

```diff
@@ -1,18 +1,15 @@
 require("dotenv").config();
 const express = require("express");
-const {
-  initializeApp,
-  applicationDefault,
-  cert,
-} = require("firebase-admin/app");
+const { initializeApp, cert } = require("firebase-admin/app");
 const firebase = require("firebase/app");
 const app = express();
 const port = process.env.PORT || 5000;
 const indexRouter = require("./routes/routes");
 const authRouter = require("./routes/authRouter");
 const eventRouter = require("./routes/eventRoutes");
-
+const verifyToken = require("./middlewares/verifyToken");
 const cors = require("cors");
+const serviceAccount = require("./adminCredentials.json");
```

Comment:
I believe this is already in the environment variables, or are we also using files with sensitive info?